g11n-ws/vip-manager-l10n-agent/src/main/resources/application.properties

```diff
@@ -25,6 +25,10 @@ remote.l10n.base.user-agent=sgtn_l10n_agent
 vip.i18n.base.url=https://localhost:8090
 #sync source to l10n manager
 vip.l10n.base.url=https://localhost:8088
+#sync batch configuration
+vip.sync.batch.enable=true
+vip.sync.batch.size=50
```

Comment:
does the size mean the number of components?

Reply:
it means the number of keys